Server/tests/controllers/checkController.test.js

```diff
@@ -9,9 +9,10 @@ import {
 import jwt from "jsonwebtoken";
 import { errorMessages, successMessages } from "../../utils/messages.js";
 import sinon from "sinon";
-describe("Check Controller - createCheck", () => {
+describe("Check Controller - createCheck", function() {
```

Comment:
What's the reasoning for replacing the arrow functions with traditional funciton expression?